index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+      <div class="nav">
+        <header>
+            <div><a href="#" class="logo"><h1>Welcome</h1></a></div>
+        </header>
+        <nav>
+                
+            <div class="Togglebutton"><a href="#"><img class="MenuIcons" src="images/icons/icon-Menu.png" alt="togglebutton"></a></div>
+        </nav>
+      </div> 
+      <main>
+        <section class="section-1">
+            <div class="Primary"><h2 id="Primary-1">Hey there. I’m <br>Mariane</h2>
+                 <h2 id="Primary-2" >I’m a software developer</h2></div>
+            <div class="supporting"> <p>I can help you build a product , feature or website Look through some of my work 
+                 and experience! If you like what you see and have a project you need coded,<br> 
+                 don’t hestiate to contact me.</p></div>
+        
+                 <ul class=social>
+                    <li><div class="social-icon"><a href="#"><img src="images/Icons/twitter.png" alt="Twitter icon"></a></div></li>
+                    <li><div class="social-icon"><a href="#"><img src="images/Icons/Linkedinicon.png" alt="Linkedin icon"></a></div></li>
```

Comment:
**Optional**

- [ ] I will suggest you add the `Linkedinicon.png` icon to the images folder. So the browser can grab and load it. This is what we have currently:

![image](https://user-images.githubusercontent.com/51296741/143587250-5bc1341b-be99-4ba1-8204-b92dc7f940af.png)